lib/output/utils.js

```diff
@@ -18,13 +18,24 @@ function define(target, source) {
   }
 }
 
+function newObjectInRealm(globalObject, object) {
+  const ctorRegistry = initCtorRegistry(globalObject);
+  return Object.defineProperties(
+    Object.create(ctorRegistry["%Object%"].prototype),
```

Comment:
nit: could we additionally capture `%Object.prototype%` in ctorRegistry, and use that here? One can imagine some weird client code overwriting `Object.prototype`.

Reply:
Good catch!